src/main/java/dev/jbang/cli/Info.java

```diff
@@ -117,6 +117,7 @@ static class ScriptInfo {
 		String gav;
 		String module;
 		Map<String, List<ProjectFile>> docs;
+		String error;
```

Comment:
I know we don't have multiple errors at this moment but I do think we will so how about we make this one an array/list and ensure we dont break parsing.